lib/msal-browser/src/encode/Base64Decode.ts

```diff
@@ -21,6 +21,19 @@ export function base64Decode(input: string): string {
  * @param base64String
  */
 function base64DecToArr(base64String: string): Uint8Array {
-    const binString = atob(base64String);
+    let encodedString = base64String.replace(/-/g, "+").replace(/_/g, "/");
+    switch (encodedString.length % 4) {
+        case 0:
+            break;
+        case 2:
+            encodedString += "==";
+            break;
+        case 3:
+            encodedString += "=";
+            break;
+        default:
+            throw new Error("Invalid base64 string");
```

Comment:
Can we make this a first class error? e.g. a BrowserAuthError?

Reply:
Done